<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortSelectDialog</class>
 <widget class="QDialog" name="PortSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>207</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Port Device</string>
  </property>
  <widget class="QPushButton" name="buttonOK">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>12</y>
     <width>91</width>
     <height>230</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioTty1">
      <property name="text">
       <string>ttyS0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioTty2">
      <property name="text">
       <string>ttyS1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioTty3">
      <property name="text">
       <string>ttyS2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioTty4">
      <property name="text">
       <string>ttyS3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioTty5">
      <property name="text">
       <string>ttyUSB0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioTty6">
      <property name="text">
       <string>ttyUSB1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioTty7">
      <property name="text">
       <string>ttyUSB2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioTty8">
      <property name="text">
       <string>ttyUSB3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioTty9">
      <property name="text">
       <string>tty000</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
